charts/istio-authorizer/values.yaml

```diff
@@ -0,0 +1,81 @@
+## istio-authorizer image parameters
+##
+image:
+  ## Image repository
+  ##
+  repository: acp.artifactory.cloudentity.com/istio-authorizer
+
+  ## Image pull policy
+  ##
+  pullPolicy: IfNotPresent
+
+  ## Image tag (immutable tags are recommended, default is the chart appVersion)
+  ##
+  tag: ""
+
+## istio-authorizer image registry secret names as an array
+##
+imagePullSecrets:
+  - name: artifactory
+
+## String to partially override istio-authorizer.name
+##
+#nameOverride: ""
+
+## String to fully override istio-authorizer.fullname
+##
+# fullnameOverride: ""
+
+## Service account to use by the istio-authorizer
+##
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+## Issuer URL
+##
+issuerURL: "http://localhost:8443/default/system"
+
+## Secret with client_id and client_secret
+##
+clientSecret:
+  # Specifies whether a secret should be created
+  create: true
+  # The name of the client secret to use
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+  # Default values
+  clientID: "YnZnN2h2YzA1NnV0YmdpaHF1OWcK"
+  clientSecret: "aE0wdmYwZHNBMHkxSHBHZW1WXzlDa21iYmlOclp2NW92S1JDTm5hd2pJbwo="
+
+## istio-authorizer ConfigMap entries
+##
+data:
+  # root CA used by istio-authorizer http client
+  ca.crt: |
```

Comment:
We should probably consider adding integration with cert-manager / cert-issuer.
i.e. allow using a certificate generated by the provided cert-issuer.

This would allow to simplify the process where ACP and istio-authorizer are deployed within one cluster.

Reply:
I believe that `root-ca: /data/ca.crt` is needed only when issuerURL uses invalid or self-signed certificate. This could happen on local dev setup but not in production. Let me know if I misunderstood it.